docs/Contribution.md

```diff
@@ -82,7 +82,7 @@ When you want to start contributing, you should [follow the instructions](https:
 
 Set your cloned fork to track upstream changes (changes to the main repository), then fetch and merge from the upstream branch:
 
-    `$ git remote add --track upstream https://github.com/systers/macc.git`
```

Comment:
@anweshknayak  It is a way to keep up to date with Upstream(Parent) repo. When you do a `rebase`, it will keep track of the branch, so you don't have to type `git rebase upstream/master`. `git rebase upstream`, will track and take care of it.